src/Comment.js

```diff
@@ -0,0 +1,65 @@
+import React, {Component} from 'react';
+import './Comment.css';
+
+class Comment extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      id: props.commentId,
+      by: null,
+      text: null,
+      time: null,
+      error: null
+    };
+
+    this.componentDidMount = this.componentDidMount.bind(this);
+    this.getData = this.getData.bind(this);
+  }
+
+  getData() {
+    fetch(`https://hacker-news.firebaseio.com/v0/item/${this.state.id}.json`)
```

Comment:
This would also be a good config thing. Nice use of templates

Reply:
I added a `config.json` and am now using it in all `fetch()` calls.